lib/Parser/ParseFlags.h

```diff
@@ -11,7 +11,7 @@ enum
     // Unused = 1 << 0,
     fscrReturnExpression = 1 << 1,   // call should return the last expression
     fscrImplicitThis = 1 << 2,   // 'this.' is optional (for Call)
-    fscrImplicitParents = 1 << 3,   // the parents of 'this' are implicit
```

Comment:
It looks like parser did not set this flag at all.

Reply:
Right. It's set by the caller of the parser, which is the thing that has the context to set it.

Reply:
Yes, - to map SCRIPTPROC_IMPLICIT_PARENTS .  (as in [https://docs.microsoft.com/en-us/scripting/winscript/reference/iactivescriptparseprocedure32-parseproceduretext](https://docs.microsoft.com/en-us/scripting/winscript/reference/iactivescriptparseprocedure32-parseproceduretext))

I am not sure though how it is different from SCRIPTPROC_IMPLICIT_THIS or whether that matters nowdays.

---
In reply to: [185670571](https://github.com/Microsoft/ChakraCore/pull/5086#discussion_r185670571) [](ancestors = 185670571)

Reply:
The two flags are treated interchangeably. I've wanted to merge them for a long time, and I'm doing it now in advance of another change involving the parser flags.